ajax/libs/zepto/package.json

```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "!(zepto).js"
```

Comment:
@hare1039 If we don't need `zepto.js` or `zepto.min.js`, `"!(zepto)*"` would be more flexible.